code/procedures/change-dwi-run-to-acq_fix_all.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# this script changes the dwi-file-names
+# for example from run-120dir to acq-120dir.
+for old_file in sub-DTA*/*/dwi/*run* ; do
```

Comment:
Where does the "DTA" in "sub-DTA*" come from? Is that really how the subjects are named? What about anonymized subject identifiers? Will they also contain "DTA"? And if so - why? This procedure is to be called after initial conversion, so "sub-*" should be sufficient.